### ISSUE TITLE
lib/text/ttml_text_parser.js: missing require: 'shaka.util.XmlUtils'

### ISSUE THREAD

**Issue by hochhaus** [google/shaka-player#1658]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**:

Yes

**What version of Shaka Player are you using?**:

v2.4.5

**Can you reproduce the issue with our latest release version?**:

Yes

**Can you reproduce the issue with the latest code from `master`?**:

Yes

**Are you using the demo app or your own custom app?**:

Custom app

**If custom app, can you reproduce the issue using our demo app?**:

N/A

**What browser and OS are you using?**:

N/A

**What are the manifest and license server URIs?**:

N/A

**What did you do?**

When attempting to compile `shaka-player` with `closure-compiler` I get the following error:

```
INFO: Analysed 240 targets (1 packages loaded, 4 targets configured).
INFO: Found 240 targets...
ERROR: /home/ahochhaus/sg/third_party/com_github_google_shaka_player/BUILD:17:1: Checking 125 JS files in //third_party/com_github_google_shaka_player:shaka_lib failed (Exit 1)
bazel-out/k8-opt/genfiles/third_party/com_github_google_shaka_player/lib/text/ttml_text_parser.js:692: ERROR - missing require: 'shaka.util.XmlUtils'
  const XmlUtils = shaka.util.XmlUtils;
                   ^
  ProTip: "JSC_MISSING_REQUIRE_STRICT_WARNING" or "strictMissingRequire" can be added to the `suppress` attribute of:
  //third_party/com_github_google_shaka_player:shaka_lib

1 error(s), 0 warning(s)

INFO: Elapsed time: 2.448s, Critical Path: 1.10s, Remote (0.00% of the time): [queue: 0.00%, setup: 0.00%, process: 0.00%]
INFO: 1 process: 1 processwrapper-sandbox.
FAILED: Build did NOT complete successfully
```

**What did you expect to happen?**

The compile to complete successfully.

**What actually happened?**

The compile failed due to a missing require.

I will happily send a pull request but my guess is that is actually more work for you to pull a one line fix into the internal perforce repo. Please let me know if you prefer a PR.

**Comment by joeyparrish** [comment]
Thanks for the report!  We'd be happy to take a pull request for this.  This team doesn't use perforce at all, and merging PRs is no big deal.  We're 100% git.  :-)

**Comment by joeyparrish** [comment]
Fix cherry-picked for v2.4.6.
